developers/academy/zero_to_mvp/101_hello_weaviate/40_set_up.mdx

```diff
@@ -18,94 +18,103 @@ import ReactPlayer from 'react-player/lazy'
 
 ## <i class="fa-solid fa-square-chevron-right"></i>&nbsp;&nbsp;Options for running Weaviate
 
-You can run Weaviate as a managed service (Weaviate Cloud Services, or WCS), or manage your own instances using Docker, Kubernetes, or Embedded Weaviate.
+Weaviate Cloud Services (WCS) can run Weaviate for you as a managed service. You can also manage your own Weavaite instances using Docker, Kubernetes, or Embedded Weaviate.
```

Comment:
I think we generally talk about WCS as a product not as an autonomous entity. Something else for the style guide perhaps.

I see your first `Weavaite` typo! You're definitely one of us now.